pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
 
 [project.optional-dependencies]
 forking-recommended = ["ujson"]
+jupyter-testing = ["jupyter_server", "ipython", "nest_asyncio"]
```

Comment:
i'd rather we put this in a "test requirements" (or just throw them into `dev-requirements.txt`)

Reply:
Then we don't need a separate job as we agreed, right?